packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -489,11 +489,8 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
             }
             if (node != null) {
               information.writeln('The nearest ancestor providing an unbounded width constraint is:');
-              information.writeln('  $node');
-              final List<DiagnosticsNode> description = <DiagnosticsNode>[];
-              node.debugFillProperties(description);
-              for (DiagnosticsNode line in description)
-                information.writeln('  $line');
+              information.write('  ');
+              information.write(node.toStringShallow('\n  '));
```

Comment:
Turns out this hides a regression: we're now missing a newline at the end of the ancestor print out, and the "See also" on the next line ends up on the last line:

```
...
I/flutter ( 4533): The nearest ancestor providing an unbounded width constraint is:
I/flutter ( 4533):   _RenderSingleChildViewport#76a9a relayoutBoundary=up10 NEEDS-LAYOUT NEEDS-PAINT
I/flutter ( 4533):   creator: _SingleChildViewport ← _ScrollableScope ← IgnorePointer-[GlobalKey#4a64a] ← Semantics ←
I/flutter ( 4533):   Listener ← _GestureSemantics ←
I/flutter ( 4533):   RawGestureDetector-[LabeledGlobalKey<RawGestureDetectorState>#20712] ←
I/flutter ( 4533):   _ExcludableScrollSemantics-[GlobalKey#fc34f] ← RepaintBoundary ← CustomPaint ← RepaintBoundary ←
I/flutter ( 4533):   NotificationListener<ScrollNotification> ← ⋯
I/flutter ( 4533):   parentData: <none> (can use size)
I/flutter ( 4533):   constraints: BoxConstraints(0.0<=w<=484.0, 0.0<=h<=920.0)
I/flutter ( 4533):   layer: OffsetLayer#77c9c
I/flutter ( 4533):   size: Size(484.0, 920.0)See also: https://flutter.io/layout/
I/flutter ( 4533): If this message did not help you determine the problem, consider using debugDumpRenderTree():
I/flutter ( 4533):   https://flutter.io/debugging/#rendering-layer
I/flutter ( 4533):   http://docs.flutter.io/flutter/rendering/debugDumpRenderTree.html
...
```

Reply:
I'll submit a PR to fix it.